### ISSUE TITLE
Allow multiple measurements and capture a derived quantity from them

### ISSUE THREAD

**Issue by grezniczek** [davwillev/stopwatch#11]
This could be
- minimum
- maximum
- average
of time elapsed.

UI could (optionally?) allow to remove individual measurements before storing the result.

**Comment by davwillev** [comment]
I like the option to remove a captured measurement before storing the result, although this does allow for potential unwanted data editing.
Calculating maxima, minima and averaging would only work with laps, but is obviously data analysis rather than collection. Maybe see if users want this?

**Comment by grezniczek** [comment]
I was thinking of tests where the instructions say: Measure 3 times, and record the best/worst/... value. And having the _holding tablet with a REDCap instrument in hand_ scenario in mind.

I guess one has to always find a compromise between (excessive) documentation and (minimally) necessary data.

**Comment by grezniczek** [comment]
Furthermore, if the UI is too strict, then issues like "Oops, I mispressed. Need to start over" become quite a nuiscance (leading in turn to potentially bad data quality and unhappy data collectors).

**Comment by davwillev** [comment]
Yes, having to start over due to a mispress is worth taking into account. Wasted data is arguably worse than missing data

**Comment by davwillev** [comment]
Maybe a user could mark for deletion with a checkbox and then confirm this. The system could maybe log the deletion to avoid the data editing issue

**Comment by grezniczek** [comment]
This woud pose the question: where to store these information? In the logs (like the field history). While the data is there, it's not really useful for analysis (and very hard to get to).

This will have to something that does indeed use a repeating instrument for storing

It is fascinating how complex a seemingly innocent thing like a stopwatch can get :)
